test/Experimental/PBWDeformations/LieAlgebras/LinearLieAlgebra-test.jl

```diff
@@ -0,0 +1,124 @@
+    ) ==
+      "3×3 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n [(-1, 2)]  [(1, 1)]   []\n [(-1, 3)]  []         [(1, 1)]\n []         [(-1, 3)]  [(1, 2)]"
+    @test repr(
+      "text/plain", liealgebra_module_struct_const(L, exterior_power(standard_module(L), 2))
+    ) ==
+      "3×3 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n []         [(-1, 3)]  [(1, 2)]\n [(1, 3)]   []         [(-1, 1)]\n [(-1, 2)]  [(1, 1)]   []"
+
+    L = special_orthogonal_liealgebra(QQ, 4)
+    @test repr("text/plain", liealgebra_struct_const(L)) ==
+      "6×6 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n []         [(-1, 4)]  [(-1, 5)]  [(1, 2)]   [(1, 3)]   []\n [(1, 4)]   []         [(-1, 6)]  [(-1, 1)]  []         [(1, 3)]\n [(1, 5)]   [(1, 6)]   []         []         [(-1, 1)]  [(-1, 2)]\n [(-1, 2)]  [(1, 1)]   []         []         [(-1, 6)]  [(1, 5)]\n [(-1, 3)]  []         [(1, 1)]   [(1, 6)]   []         [(-1, 4)]\n []         [(-1, 3)]  [(1, 2)]   [(-1, 5)]  [(1, 4)]   []"
+    @test repr(
+      "text/plain",
+      liealgebra_module_struct_const(L, symmetric_power(standard_module(L), 1)),
+    ) ==
+      "6×4 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n [(-1, 2)]  [(1, 1)]   []         []\n [(-1, 3)]  []         [(1, 1)]   []\n [(-1, 4)]  []         []         [(1, 1)]\n []         [(-1, 3)]  [(1, 2)]   []\n []         [(-1, 4)]  []         [(1, 2)]\n []         []         [(-1, 4)]  [(1, 3)]"
+    @test repr(
+      "text/plain", liealgebra_module_struct_const(L, exterior_power(standard_module(L), 1))
+    ) ==
+      "6×4 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n [(-1, 2)]  [(1, 1)]   []         []\n [(-1, 3)]  []         [(1, 1)]   []\n [(-1, 4)]  []         []         [(1, 1)]\n []         [(-1, 3)]  [(1, 2)]   []\n []         [(-1, 4)]  []         [(1, 2)]\n []         []         [(-1, 4)]  [(1, 3)]"
+    @test repr(
+      "text/plain", liealgebra_module_struct_const(L, exterior_power(standard_module(L), 2))
+    ) ==
+      "6×6 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n []         [(-1, 4)]  [(-1, 5)]  [(1, 2)]   [(1, 3)]   []\n [(1, 4)]   []         [(-1, 6)]  [(-1, 1)]  []         [(1, 3)]\n [(1, 5)]   [(1, 6)]   []         []         [(-1, 1)]  [(-1, 2)]\n [(-1, 2)]  [(1, 1)]   []         []         [(-1, 6)]  [(1, 5)]\n [(-1, 3)]  []         [(1, 1)]   [(1, 6)]   []         [(-1, 4)]\n []         [(-1, 3)]  [(1, 2)]   [(-1, 5)]  [(1, 4)]   []"
+    @test repr(
+      "text/plain", liealgebra_module_struct_const(L, exterior_power(standard_module(L), 3))
+    ) ==
+      "6×4 Matrix{Vector{Tuple{QQFieldElem, Int64}}}:\n []         []         [(-1, 4)]  [(1, 3)]\n []         [(1, 4)]   []         [(-1, 2)]\n [(-1, 4)]  []         []         [(1, 1)]\n []         [(-1, 3)]  [(1, 2)]   []\n [(1, 3)]   []         [(-1, 1)]  []\n [(-1, 2)]  [(1, 1)]   []         []"
+
+    L = special_orthogonal_liealgebra(QQ, 5)
+    @test repr(
+      "text/plain", liealgebra_module_struct_const(L, exterior_power(standard_module(L), 1))
```

Comment:
Any particular reason to test the *text representation* of the output, instead of structurally checking the output, i.e., comparing it to a honest Julia object?

Reply:
ehm, probably some historical reason for different (non-oscar) internal types that does not hold anymore. will be fixed